### PR TITLE
TIC1

### PR DESCRIPTION
**Pull request by timohahaaa**
https://timohahaaa.github.io/layout_moyo-header/report/html_report/

https://timohahaaa.github.io/layout_moyo-header/

I'm so sorry, but I created a new pull request cause the last one had a glitch on my PC!

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,65 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:3]: there is no default padding for body, so there is no need to reset it

File: src/style.css [hunk 2]
```diff
@@ -1 +1,65 @@
...
+  height: 40px;
+  margin: 10px 50px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/style.css:29]: just set it for html selector and inherit it

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]
Pls try to fix failed report points. If you have any troubles - feel free to ask them in the Slack

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,40 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          src="images/logo.png"
+          alt="#"
```
Comment (polosanya) [inline src/index.html:20]: The `alt` attribute for the logo image should descriptive. Something like "Moyo logo" would be great

File: src/style.css [hunk 4]
```diff
@@ -1 +1,69 @@
...
+
+.nav {
+  display: flex;
+  margin-right: 30px;
+}
+
+.nav__link {
+  font-family: inherit;
+  position: relative;
+  align-items: center;
```
Comment (polosanya) [inline src/style.css:33]: The 'align-items' property is used with a flex container, but '.nav__link' is not a flex container. Consider removing it, as it does not have any effect on the element

File: src/style.css [hunk 5]
```diff
@@ -1 +1,69 @@
...
+  left: 0;
+  top: 34px;
+}
+
+.nav__item {
+  display: inline-block;
+  list-style-type: none;
+  margin-left: 20px;
+  padding-top: 19px;
+  padding-bottom: 19px;
```
Comment (polosanya) [inline src/style.css:68]: Paddings should be added to the links, not li, so links are clickable for the whole height

**Review by polosanya — CHANGES_REQUESTED** [review]
A few more comments to fix

**Pull request closed without merging**